samples/server/petstore/rust-server/output/openapi-v3/api/openapi.yaml

```diff
@@ -640,7 +640,6 @@ components:
           nullable: true
           type: string
         nullableWithNullDefault:
-          default: "null"
```

Comment:
@benzvan thanks for the PR. Is this a regression? Did the changelog mention anything related to this change?

Reply:
That must have crept in when I ran the generator scripts. I didn't notice any output related to it.

Reply:
Though reading it maybe it's just being more explicit? Seems like it's reasonable for a `nullableWithNullDefault` to have a default of `null`

Reply:
I'm not familiar enough with the upstream project to speak intelligently to the change though. I just know it used to use a 4-year-old version of httpclient. :-D

Reply:
@wing328 any followup thoughts? The only line I manually changed was the version number for swagger-parser.

Reply:
Poking around the swagger-parser notes I see this _could_ be related but I don't know enough about the code to be sure.
https://github.com/swagger-api/swagger-parser/pull/1600/files